R/render.R

```diff
@@ -67,6 +67,28 @@ render_page_html <- function(pkg, name, data = list(), depth = 0L) {
   )
   components <- purrr::map(templates, render_template, data = data)
   components <- purrr::set_names(components, pieces)
+
+  # Special case for the navbar that uses a nested partial template
```

Comment:
I don't really want to make any exceptions to the templating system as it makes the whole thing harder to reason about. I was thinking that instead it would be better to pull more out in to yaml so that you can set it there.

Reply:
But could that YAML part use variables? (e.g. the package name and version)

Reply:
We could set it up like the other navbar pieces, i.e. have package name and version components that default to the package name and version respectively?

Reply:
Ah ok, I'll make a new PR!